api/.babelrc

```diff
@@ -1,7 +1,14 @@
 {
-    "presets": ["es2015-node6", "es2016", "stage-2"],
+    "presets": [
+      ["env", {
+          "targets": {
+              "node": "current"
```

Comment:
For now, AWS Lambda can only be run with Node 6.10.

![image](https://user-images.githubusercontent.com/1819833/32717537-457ad14e-c85a-11e7-9579-573e16e8d2d2.png)

Can you configure babel in order to support this version, wherever we transpile the API?